src/frontend_cache/cache.c

```diff
@@ -25,9 +54,46 @@ void* search_fast_cache(size_t size) {
     return NULL;
 }
 
+void* search_unsorted_cache(size_t size) {
+    t_chunk* unsorted_cache = g_pagemap->frontend_cache->unsorted_cache;
```

Comment:
definitely more work to do in this function, as you were saying it should sort chunks into the cache_table!

I'll spend more time this weekend to look into that work